scripts/oes-data-migration-scripts/migration_v4.0.3_to_v4.0.4.yaml

```diff
@@ -0,0 +1,139 @@
+    app.kubernetes.io/managed-by: Helm
+    chart: oes-4.0.9
+    component: auditservice
+    heritage: Helm
+  name: oes-audit-service-new
+spec:
+  internalTrafficPolicy: Cluster
+  ipFamilies:
+  - IPv4
+  ipFamilyPolicy: SingleStack
+  ports:
+  - name: auditservice
+    port: 8097
+    protocol: TCP
+    targetPort: 8097
+  selector:
+    app: oes
+    component: auditservice-new
+  sessionAffinity: None
+  type: ClusterIP
+status:
+  loadBalancer: {}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: oes
+    app.kubernetes.io/managed-by: Helm
+    chart: oes-4.0.9
+    component: auditservice-new
+    heritage: Helm
+  name: oes-audit-service-new
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: oes
+      component: auditservice-new
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+  template:
+    metadata:
+      annotations:
+        moniker.spinnaker.io/application: isd
+        prometheus.io/scrape: "false"
+        prometheus_io_path: /mgmt/prometheus
+        prometheus_io_port: "8097"
+      labels:
+        app: oes
+        chart: oes-4.0.9
+        component: auditservice-new
+        heritage: Helm
+    spec:
+      containers:
+      - image: quay.io/opsmxpublic/ubi8-oes-audit-service-migration:v403-v404
+        imagePullPolicy: IfNotPresent
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /mgmt/health
+            port: 8097
+            scheme: HTTP
+          initialDelaySeconds: 60
+          periodSeconds: 60
+          successThreshold: 1
+          timeoutSeconds: 1
+        name: oes-audit-service-new
+        ports:
+        - containerPort: 8097
+          name: backend
+          protocol: TCP
+        readinessProbe:
+          failureThreshold: 3
+          initialDelaySeconds: 60
+          periodSeconds: 10
+          successThreshold: 1
+          tcpSocket:
+            port: 8097
+          timeoutSeconds: 1
+        resources: {}
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        volumeMounts:
+        - mountPath: /opsmx/conf/audit-service-local.yml
+          name: audit-config-volume
+          subPath: audit-local.yml
+        - mountPath: /opsmx/conf/bootstrap.yml
+          name: bootstrap-config-volume
+          subPath: bootstrap.yml
+        - mountPath: /opsmx/conf/standard-error-code.csv
+          name: standard-error-conf
+          subPath: standard-error-codes.csv
+      dnsPolicy: ClusterFirst
+      restartPolicy: Always
+      schedulerName: default-scheduler
+      securityContext: {}
+      terminationGracePeriodSeconds: 30
+      volumes:
+      - name: audit-config-volume
+        secret:
+          defaultMode: 420
+          items:
+          - key: audit-local.yml
+            path: audit-local.yml
+          secretName: oes-audit-service-config-new
+      - name: bootstrap-config-volume
```

Comment:
is this required ??

Reply:
Checked with developer(Rahul) - those 2 are required.